cmd/scollector/collectors/haproxy_unix.go

```diff
@@ -0,0 +1,518 @@
+package collectors
+
+import (
+	"encoding/csv"
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+var trackedInstances = []*trackedInstance{}
+
+type trackedInstance struct {
+	Tier     string
+	URL      string
+	Username string
```

Comment:
Why does each instance have a username and password? You are already passing the un/pw to most methods, just use it via parameters rather than saving it to each instance struct.